test/collection/test_validator.py

```diff
@@ -0,0 +1,38 @@
+from typing import Any, List
+
+import numpy as np
+import pandas as pd
+import polars as pl
```

Comment:
Since we have the deps available at testing time, can we add these different package array types to the `test_near_vector_with_other_input` above too or do you feel they're covered enough by `test_validator` below?

Reply:
added :)